packages/contracts/src/predeploys.ts

```diff
@@ -3,9 +3,9 @@
  * various useful functions.
  *
  * Notes:
- * - 0x42...04 was the address of the OVM_ProxySequencerEntrypoint. This contract is no longer in
- *    use and has therefore been removed. We may place a new predeployed contract at this address
- *    in the future. See https://github.com/ethereum-optimism/optimism/pull/549 for more info.
+ * 0x42...04 was the address of the OVM_ProxySequencerEntrypoint. This contract is no longer in
```

Comment:
what rule is this? I kind of like using indentation in comments to delineate sections.

Reply:
`jsdoc/check-indentation` 

More than happy to remove tho if this is the preference

Reply:
Yeah I think it would be better to remove this rule.